### PR TITLE
Allow "=" in config values 

### PR DESCRIPTION
**Pull request by quantumagi**
After the PR we will be able to pass e.g. Azure keys as command line arguments. 

These mime-encoded-data strings end in one or more "=" - e.g.:

`azurekey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==`

### CODE REVIEW

File: Stratis.Bitcoin/Configuration/TextFileConfiguration.cs [hunk 1]
```diff
@@ -39,10 +39,11 @@ public TextFileConfiguration(string[] args)
             this.args = new Dictionary<string, List<string>>();
             foreach (string arg in args)
             {
+                // Split on the FIRST "=".
                 string[] splitted = arg.Split('=');
-                if (splitted.Length == 2)
-                    this.Add(splitted[0], splitted[1]);
-                if (splitted.Length == 1)
+                if (splitted.Length > 1)                    
+                    this.Add(splitted[0], string.Join("=", splitted.Skip(1)));
```
Comment (bokobza) [inline Stratis.Bitcoin/Configuration/TextFileConfiguration.cs]: Can you please add a little comment here to show what kind of value this helps parse?

**Comment by bokobza** [comment]
Nice one @quantumagi ! Can you please add a couple of unit tests to this?

File: Stratis.Bitcoin/Configuration/TextFileConfiguration.cs (hunk 1 above)
Comment (quantumagi) [reply Stratis.Bitcoin/Configuration/TextFileConfiguration.cs]: Done.

**Comment by quantumagi** [comment]
Unit tests have been added.

**Comment by bokobza** [comment]
Very nice, thank you

**Pull request merged**